armi/tests/anl-afci-177/anl-afci-177-fuelManagement.py

```diff
@@ -22,9 +22,8 @@ def chooseSwaps(self, shuffleParameters):
             self.r.p.cycleLength * self.r.p.availabilityFactor * units.SECONDS_PER_DAY
         )
         for a in self.r.core:
-            peakFluence = a.getMaxParam("fastFluence")
             peakFlux = a.getMaxParam("fastFlux")
-            if peakFluence + peakFlux * cycleSeconds > 4.0e23:
```

Comment:
The argument here being that peakFluence was always zero and therefore doesn't need to be included?

Reply:
yup!